viewConfig.js

```diff
@@ -26,10 +26,14 @@ const DEFAULT_THROTTLE_MILLISECS = 4000;
 const realtimeEventThrottleMilliseconds =
  pe.realtimeEventThrottleMilliseconds || DEFAULT_THROTTLE_MILLISECS;
 
+const socketIOtransportProtocol = pe.SOCKETIO_TRANSPORT_PROTOCOL || 'undefined';
```

Comment:
Why using string 'undefined' here? It seems like it has potential for misunderstanding given javascript's "undefined" reserved word. I would rather see something like this:

`const socketIOtransportProtocol = pe.SOCKETIO_TRANSPORT_PROTOCOL || null;`

or

`const socketIOtransportProtocol = pe.SOCKETIO_TRANSPORT_PROTOCOL || '';`

... and then in app.js line 105, just check for `if (clientProtocol) {`